modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web/src/main/resources/META-INF/resources/admin/process/workflow_action.jsp

```diff
@@ -62,8 +62,10 @@ String backURL = (String)row.getParameter("backURL");
 		<portlet:param name="version" value="<%= String.valueOf(workflowDefinition.getVersion()) %>" />
 	</portlet:actionURL>
 
-	<liferay-ui:icon
-		message="unpublish"
-		url="<%= deactivateWorkflowDefinition %>"
-	/>
+	<c:if test="<%= permissionChecker.isCompanyAdmin() %>">
+		<liferay-ui:icon
```

Comment:
Hey @seltong, please move the `<portlet:actionURL ...> ...` declaration to the if statement as well.

Thx